xml/System/IntPtr.xml

```diff
@@ -537,15 +539,15 @@
         <Parameter Name="offset" Type="System.Int32" Index="1" FrameworkAlternate="dotnet-uwp-10.0;net-5.0;net-6.0;net-7.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-1.0;netstandard-1.1;netstandard-1.2;netstandard-1.3;netstandard-1.4;netstandard-1.5;netstandard-1.6;netstandard-2.0;netstandard-2.1;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
       </Parameters>
       <Docs>
-        <param name="pointer">The pointer to add the offset to.</param>
+        <param name="pointer">The signed integer to add the offset to.</param>
         <param name="offset">The offset to add.</param>
-        <summary>Adds an offset to the value of a pointer.</summary>
-        <returns>A new pointer that reflects the addition of <paramref name="offset" /> to <paramref name="pointer" />.</returns>
+        <summary>Adds an offset to a signed integer.</summary>
+        <returns>A new signed integer that reflects the addition of <paramref name="offset" /> to <paramref name="pointer" />.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The <xref:System.IntPtr.Add%2A> method does not throw an exception if the result is too large to represent as a pointer on the specified platform. Instead, the addition operation is performed in an unchecked context.
+ The <xref:System.IntPtr.Add%2A> method does not throw an exception if the result is too large to represent as a signed integer in the executing process. Instead, the addition operation is performed in an unchecked context.
```

Comment:
It might be worth a comment calling out that `Add` and `Subtract` aren't recommended for use from C# or F#. Rather devs should prefer using the direct operators (`+` and `-`) instead and to make use of `checked` and `unchecked` contexts depending on their needs.